cubed/tests/test_html.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from cubed.array_api import ones
+
+
+def test_repr_html():
+    pytest.importorskip("jinja2")
+    assert ones([])._repr_html_()
+    assert ones(10)[:0]._repr_html_()
+    assert ones(10)._repr_html_()
+    assert ones((10, 10))._repr_html_()
+    assert ones((10, 10, 10))._repr_html_()
+    assert ones((10, 10, 10, 10))._repr_html_()
```

Comment:
Also copied from dask. (Do we need to attribute it specifically?)

Surprisingly this seems to be the only test dask has for the array repr specifically.

Reply:
> Also copied from dask. (Do we need to attribute it for license purposes?)

How about adding a comment referring to the license in vendor/dask.